prelude/README.md

```diff
@@ -118,6 +139,16 @@ instance categoryArr :: Category Prim.Function
 (#) :: forall a b. a -> (a -> b) -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+    (((3 + 1) * 2) + 4) = 3 # ((+) 1) # ((*) 2) # ((+) 4)
```

Comment:
Like above, we should not use sections in these examples, and they should be more understandable. Would you want to read documentation like this?